lesson_17/dags/sql/transfer_sales_bronze_to_silver.sql

```diff
@@ -0,0 +1,47 @@
+DECLARE v_dag_logical_date DEFAULT CAST(CAST('{{ dag_run.logical_date }}' AS TIMESTAMP) AS DATE);
+DECLARE v_dag_start_date DEFAULT CAST('{{ dag_run.start_date }}' AS TIMESTAMP);
+DECLARE v_dag_ds DEFAULT CAST(CAST('{{ ds }}' AS TIMESTAMP) AS DATE);
+
+DELETE FROM `{{ params.project_id }}.silver.sales`
+WHERE purchase_date = v_dag_ds
+;
+
+INSERT `{{ params.project_id }}.silver.sales` (
+    client_id,
+    purchase_date,
+    product,
+    price,
+    currency,
+
+    _id,
+    _logical_dt,
+    _job_start_dt
+)
+SELECT
+    CAST(CustomerId AS INT64)       AS client_id,
+    CASE
```

Comment:
This moment is not quite trivial. I would recommend to put a comment at least here